### PR TITLE
Revise branching

### PR DESCRIPTION
**Pull request by chrissyrissmeyer**
Combines branching and merging into 1 episode.

If you are happy with it, we will need to delete merging and rename the other episodes once this PR is closed.

### CODE REVIEW

**Comment by dunn** [comment]
> we will need to delete merging and rename the other episodes once this PR is closed.

Could you add that as a commit to this PR?

File: _episodes/06-branching.md [hunk 1]
```diff
@@ -116,13 +122,21 @@ nothing to commit, working tree clean
 ~~~
 {: .bash}
 
+>Tip: If need to, you can rename a branch using the `git branch -m` (move) command. The 
+syntax is >`git branch -m old-name new-name`.
+>
+>~~~
+>$ git branch -m title-cads title-caps
```
Comment (dunn) [inline _episodes/06-branching.md]: I like this better; less chance of half of the students renaming their branch, and half of those giving it a new name they made up.

File: _episodes/06-branching.md [hunk 2]
```diff
@@ -131,158 +145,217 @@ branch.
 
 ~~~
 $ git branch
-* cleanup-plan
   master
+* title-caps
 ~~~
 {: .bash}
 
+>Tip: You can create and checkout a branch at the same time using `git checkout`
```
Comment (dunn) [inline _episodes/06-branching.md:153]: Is this what the `{: .callout}` tag is for?

**Comment by dunn** [comment]
Overall looks great!

File: _episodes/06-branching.md (hunk 2 above)
Comment (chrissyrissmeyer) [reply _episodes/06-branching.md:153]: I have no idea.

**Comment by chrissyrissmeyer** [comment]
@dunn: Glad you like what I did. I was originally worried that the credit removals would end up causing so pretty harry conflicts -- now that I am done with that work, I don't think it will be a problem. Happy to update the PR with the episode deletion and renaming.

**Comment by chrissyrissmeyer** [comment]
Okay, I have not removed the merging episode and renumbered the rest. Is there any other file I need to update (like an index file or something)?

**Comment by dunn** [comment]
Nope, the index page is generated automatically based on the episode filenames.

**Comment by dunn** [comment]
I did the rebase.

**Review by dunn — APPROVED** [review]

**Review by mcritchlow — APPROVED** [review]
LGTM!

**Pull request merged**